### PR TITLE
Add basic support for production cluster

### PR DESCRIPTION
**Pull request by peggyl**
Use the ecs-production-Cluster when env = production.

Tested locally with one app in a dev environment and one app in production.

Also productionizes by using the system ecs-metrics, which depends on https://github.com/Clever/ecs-metrics/pull/9. /cc @nathanleiby

### CODE REVIEW

File: dashboards/ecs_services.js [hunk 1]
```diff
@@ -60,39 +60,40 @@ function getErrorRow(errMsg) {
 // accessable variables in this scope
 var window, document, ARGS, $, jQuery, moment, kbn;
 
-return function(callback) {
-  // Setup some variables
-  var dashboard;
+var config = {
+  ecsMetricsURL: "https://system--ecs-metrics.int.clever.com",
```
Comment (nathanleiby) [inline dashboards/ecs_services.js]: Want to go ahead and move this to prod: "https://production--ecs-metrics.int.clever.com"?

**Comment by nathanleiby** [comment]
lgtm

File: dashboards/ecs_services.js (hunk 1 above)
Comment (peggyl) [inline dashboards/ecs_services.js]: 💯

**Comment by peggyl** [comment]
@nathanleiby updated to use production--ecs-metrics!

**Comment by nathanleiby** [comment]
👏

**Pull request merged**